magento/inventory
# #208 - Add Support of new Qty format for Product Import

### Description

Adds support for new Qty format for Product Import as seen in story for [Update Product import export to include inventory source](https://github.com/magento-engcom/msi/wiki/Update-Product-import-export-to-include-inventory-source).

### Fixed Issues (if relevant)

1. magento-engcom/mis#208: Add Support of new Qty format for Product Import 

### Manual testing scenarios

1. Create new Product Import with the new supported formats for sources. eg `qty => 1=10|2=20`
2. Check Data and Import.
3. Check the Data for the Product, new data for sources should be imported.

### Outstanding Tasks
 - [ ] Tests for new Import Processors
 - [ ] Add Import Processors for new qty field formats
 - [ ] Rework validation for Catalog Import/Export

### Contribution checklist
